katharsis-client/src/main/java/io/katharsis/client/internal/BaseResponseDeserializer.java

```diff
@@ -109,9 +110,11 @@ public void setRelations(ResourceBodies dataBodies) {
 				Object resource = resourceMap.get(uid);
 				
 				RegistryEntry registryEntry = resourceRegistry.getEntry(body.getType());
-				QueryParams queryParams = null;
+				QueryAdapter queryAdapter = null;
 				RepositoryMethodParameterProvider parameterProvider = null;
-				setRelations(resource, registryEntry, body, queryParams, parameterProvider);
+				
+				// FIXME read includes
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1134)